hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -89,7 +89,7 @@
               - verify that all speeds are supported
             '''
       milestone: V2
-      tests: []
+      tests: ["spi_host_speed"]
```

Comment:
do you test all the modes in this test?

Reply:
for debug purpose and for designer to reproduce only Dual mode is enabled in this test but the test has the knob for Quad mode also.
This will be enabled later and run with all modes together.

Reply:
@weicaiyang eventually all modes will be tested in this test. But I think @visbaz is verifying one at a time - which makes sense